CMake/SlicerBlockInstallQtTools.cmake

```
@@ -1,10 +1,12 @@
-
 # Get root directory
 get_property(_filepath TARGET "Qt5::Core" PROPERTY LOCATION_RELEASE)
 get_filename_component(_dir ${_filepath} PATH)
 if(APPLE)
   # "_dir" of the form "<qt_root_dir>/lib/QtCore.framework"
   set(qt_root_dir "${_dir}/../..")
+elseif(${_dir} MATCHES "x86_64-linux-gnu")
+  # "_dir" of the form "<qt_root_dir>/lib/x86_64-linux-gnu"
+  set(qt_root_dir "${_dir}/../../..")
 else()
   # "_dir" of the form "<qt_root_dir>/lib"
   set(qt_root_dir "${_dir}/..")
```

```suggestion
  set(qt_root_dir "${_dir}/..")
  # "_dir" of the form "<qt_root_dir>/lib/<arch>" (e.g "<qt_root_dir>/lib/x86_64-linux-gnu")
  if("${_dir}" MATCHES "${CMAKE_LIBRARY_ARCHITECTURE}")
    set(qt_root_dir "${qt_root_dir}/..")
  endif()
```

This is a more general approach allowing to support other architecture (e.g arm64). See https://cmake.org/cmake/help/v3.31/variable/CMAKE_LIBRARY_ARCHITECTURE.html#cmake-library-architecture